src/grdmask.c

```diff
@@ -616,11 +651,11 @@ EXTERN_MSC int GMT_grdmask (void *V_API, int mode, void *args) {
 
 					/* Here we will have to consider the x coordinates as well (or known_side is set) */
 #ifdef _OPENMP
-#pragma omp parallel for private(col,xx,side,ij) shared(n_columns,GMT,Grid,do_test,yy,S,known_side,row,Ctrl,z_value,mask_val)
+#pragma omp parallel for private(col,ij,xx,side,z_to_set) shared(n_columns,Grid,row,Ctrl,node_is_set,GMT,do_test,yy,S,known_side,z_value,mask_val)
```

Comment:
The new test maskmode.sh fails on Linux when OpenMP support enabled. I have not checked if it's the same on macOS, but I would assume so. There's also a compiler warning that z_to_set may be uninitialized. @PaulWessel do you notice anything obvious wrong with the OpenMP implementation here? Otherwise I can debug.